README.md

```diff
@@ -1,3 +1,15 @@
 # gen3users
 
-utils for Gen3 commons user management
+Utils for Gen3 commons user management.
+
+## user.yaml validation
+
+How to validate a `user.yaml` file:
+```
+from gen3users.validation import validate_user_yaml
+
+file_name = "my-gen3-commons/user.yaml"
+with open(file_name, "r") as f:
+    user_yaml = f.read()
+    validate_user_yaml(user_yaml, file_name)
+```
```

Comment:
Can we make this a [CLI](https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point) please?


```bash
$ pip install gen3users
$ gen3users validate my-gen3-commons/user.yaml
```